tests/Unit/ApparentHorizons/Test_Tags.cpp

```diff
@@ -281,24 +281,26 @@ void test_normals() {
   CHECK_ITERABLE_APPROX(expected_normal_mag, get(normal_mag));
 }
 
+void test_max_ricci_scalar() {
+  Scalar<DataVector> d{{{{1., 2., 3.}}}};
+  const double expected_max{max(get(d))};
```

Comment:
Just write `3.` here.  In a test, it is best to just write in the answer if it is easy to find.  This eliminates the possibility that there could be a bug in the code that calculates the expected value.